ui/components/component-library/tag/tag.types.ts

```diff
@@ -0,0 +1,17 @@
+import { TextProps } from '../text/text.types';
+import { BoxProps } from 'ui/components/ui/box/box';
```

Comment:
Ah this will fix the error you are getting
```suggestion
import type { BoxProps } from '../../ui/box/box.d';
```